### PR TITLE
Local Preview instructions

### PR DESCRIPTION
**Pull request by nastiatikk**

### CODE REVIEW

**Comment by nastiatikk** [comment]
I am not sure about these two steps:
npm install -g grunt-cli to install Grunt
npm install -g bower to install Bower

Should they be run on the first project run or it's a one-time installation like node.js?

**Comment by vmarta** [comment]
@nastiatikk they're for one-time installation like installing node.

**Comment by ericmuyser** [comment]
As denoted by the `-g` parameter (global)

**Comment by nastiatikk** [comment]
@ry5n @jeffkamo Any suggestions on this updated Contribution section?

**Comment by ry5n** [comment]
- I don’ think the changes to the “Requirements” section clarify anything. @nastiatikk if you were confused by the original, what did you find confusing?
- Same with renaming the “Steps” section to “Running”
- The new “Building Variations” heading was confusing to me, because I thought it was intended to be instructions on building new Variations (?) of Pinny. Now I guess that this is intended to document the other grunt tasks that are available with the project. A section called “Available Grunt Tasks” might be better.
- Although a section called Contributing is common for project readmes, I’m not sure that’s what this section was aiming for. The Contributing section usually contains instructions for people who want to help maintain/improve the project. What is there now seems more focused on creating a custom build as a fork (?). Dunno.

**Comment by jeffkamo** [comment]
It seems like ultimately all that was wanted was a simple "How to Preview Locally" instructions.

So I don't think it's necessary to rename the headings or add the "Building Variations" section.

Listing the grunt tasks might help, or simply having a bit of `use grunt serve to preview the site at localhost:3000` mentioned after the build steps would probably suffice.

**Comment by nastiatikk** [comment]
* 'Requirements' changes is my mistake. I changed it and then rolled back but I see I did it with mistakes
* The same with 'Running'
* I will rename title for Building Variations
* I will rename this section to 'Working with components and utilities locally', it will be more specific.

**Comment by nastiatikk** [comment]
@ry5n @jeffkamo @kpeatt @avelinet @cole-sanderson @mlworks 
I've updated pinny's instructions based on all the comments above. Please review.

File: README.md [hunk 1]
```diff
@@ -521,19 +521,31 @@ $pinny.pinny('close');
...
+
 
 ### Steps
 1. `npm install`
 1. `bower install`
-1. `grunt build`
+1. `grunt serve`
+
+
+##### Grunt Tasks:
```
Comment (jeffkamo) [inline README.md]: This can probably be just a level 3 heading, since I don't think it makes sense to be a sub-topic of "Steps"

**Comment by jeffkamo** [comment]
Would love to hear an engi's thoughts on these changes. @ericmuyser @vmarta @tedtate

**Comment by scalvert** [comment]
Made some quick changes to terminology. LGTM! 

:+1:

**Pull request merged**